src/main/java/com/znsio/e2e/tools/cmd/CommandLineExecutor.java

```diff
@@ -9,8 +9,13 @@
 
 public class CommandLineExecutor {
     private static final Logger LOGGER = Logger.getLogger(CommandLineExecutor.class.getName());
+    private static final int DEFAULT_COMMAND_TIMEOUT = 120;
```

Comment:
Given in many cases you are now passing the timeout as 60, should default itself should be 60?

Reply:
done